spec/utils-spec.js

```diff
@@ -10,7 +10,7 @@ describe('Utils Tests', () => {
 
   it('should use exec function successfully', (done) => {
     utils.exec('echo ping').then((result) => {
-      expect(result).toBe('ping\n');
+      expect(result).toBe('ping\r\n');
```

Comment:
This seems to be a Windows-specific issue. Because a huge amount of contributors (including @jywarren and myself) use *nix systems, and we use Travis over AppVeyor for testing, I think this should stay. However, you're right. We need to figure out an alternative way to make this work in a cross-platform manner (let's strip whitspaces before testing?)

Thanks for bring this to our notice.

Reply:
P.S. This is probably why Travis failed your PR.

Reply:
Thanks for the quick feedback! Inbox is being lazy, otherwise I would have seen this a lot sooner :)

I thought this might be the case and even wondered if my version of Jasmine was weird. I'll go ahead and change it back for the time being. Always happy to nudge cross-platforming!